libraries/common/providers/keyboard/index.jsx

```diff
@@ -35,15 +37,21 @@ class KeyboardConsumer extends PureComponent {
    * Stores current keyboard state.
    */
   handleKeyboardChange = ({ open, overlap, duration }) => {
+    // eslint-disable-next-line extra-rules/no-single-line-objects
```

Comment:
Please follow the rules and don't deactivate them ;) There have been created for a good reason.